query-engine/connector-test-kit-rs/query-engine-tests/src/utils/raw.rs

```diff
@@ -2,7 +2,7 @@ use chrono::{DateTime, FixedOffset};
 use prisma_value::encode_bytes;
 use query_tests_setup::{TestError, TestResult};
 
-pub fn fmt_query_raw(query: &str, params: Vec<RawParam>) -> String {
+pub fn fmt_query_raw(query: &str, params: impl IntoIterator<Item = RawParam>) -> String {
```

Comment:
I wanted to use an array and not a vec since there was no need for a vec except the function signature required it, so I made this generic. We can now remove unnecessary allocations in a bunch of tests that are using it too.